locales/hr.json

```diff
@@ -42,64 +38,57 @@
     },
     "cart": {
-        "subtotal": "Međuzbroj",
-        "shipping_taxes_calculated_at_checkout": "Dostava i porezi će se obračunati na blagajni, nakon unosa svih podataka.",
+        "subtotal": "Neto iznos",
+        "shipping_taxes_calculated_at_checkout": "PDV i eventualni troškovi dostave obračunat će se nakon unosa svih podataka.",
         "loading": "Pripremamo vašu košaricu ...",
         "secured_by": "Secured by Snipcart",
-        "summary": "Pregled Narudžbe",
+        "summary": "Pregled narudžbe",
         "empty": "Vaša košarica je prazna.",
-        "invoice_number": "Broj Ponude",
-        "view_detailed_cart": "Detalji košarice"
+        "invoice_number": "Ponuda br."
     },
     "order": {
         "loading": "Pripremamo vašu narudžbu..."
     },
     "discount_box": {
         "promo_code": "Promocijski kod?",
-        "promo_code_placeholder": "Promocijski kod",
+        "promo_code_placeholder": "Unesite promocijski kod",
         "promocode_applied": "Promocijski kod je prihvaćen"
     },
     "address_form": {
         "name": "Ime i prezime",
         "email": "Email adresa",
-        "address1": "Ulica i broj",
-        "address2": "Kućni broj, kat, apartman, broj ureda, ...",
+        "address1": "Ulica i br.",
+        "address2": "Kat, ured ...",
         "city": "Grad",
         "country": "Država",
         "postalCode": "Poštanski broj",
         "province": "Županija",
         "dont_see_address": "Ne nalazim svoju adresu"
     },
     "billing": {
-        "title": "Izdavanje računa",
+        "title": "Podaci o primatelju računa",
         "address": "Adresa na računu",
         "continue_to_shipping": "Nastaviti na dostavu",
         "use_different_shipping_address": "Koristiti drugu adresu za dostavu"
     },
     "customer": {
-        "information": "Podaci o kupcu"
-    },
-    "customer_details": {
-        "title": "Detalji o kupcu"
+        "information": "Informacije o kupcu"
     },
     "customer_dashboard": {
         "my_account": "Moj račun",
         "ordered_on": "Naručeno dana",
-        "next_billing_date": "Datum izdavanja sljedećeg Računa",
+        "next_billing_date": "Datum izdavanja sljedećeg računa",
         "price": "Cijena",
         "total": "Ukupno",
         "status": "Status",
         "order": "Ponuda %{invoice_number}",
-        "order_details": "Detalji Ponude",
+        "order_details": "Detalji narudžbe",
         "loading": "Učitavam...",
-        "no_orders": "Nema Ponuda.",
-        "no_subscriptions": "Nema pretplata.",
-        "view_invoice": "Pregled Računa",
+        "no_orders": "Narudžbe nisu pronađene.",
+        "no_subscriptions": "Pretplate nisu pronađene.",
+        "view_invoice": "Pogledati račun",
         "sign_out": "Odjava",
-        "show_more_orders": "Više %{smart_count} Narudžbi |||| Više %{smart_count} Narudžbi",
-        "show_more_subscriptions": "Više %{smart_count} Pretplata |||| Više %{smart_count} Pretplata",
```

Comment:
Please make sure you add back those phrases:

- show_more_subscriptions
- orders
- subscriptions